### DISCUSSION TITLE
Possible to re-inject mantine styles in head

### DISCUSSION THREAD

**Discussion by roykolak** [mantinedev/mantine#7146] [Q&A] [answered]
Looking for a way to re-inject mantine styles in the head after a page load. 

Background: I have a chrome extension that is using mantine and a website that uses turbo links is breaking styles due to the head getting cleared on navigation.

**Comment by rtivital** [comment, marked as answer]
Mantine does not inject styles into head, all styles are either inline or included in css file.

**Reply by roykolak** [reply]
Ah my mistake! The issue was the top level data attribute on `<html>` was getting cleared which resulted in some important css vars getting unset.
